tests/go.mod

```diff
@@ -7,7 +7,6 @@ toolchain go1.23.7
 replace (
 	go.etcd.io/etcd/api/v3 => ../api
 	go.etcd.io/etcd/client/pkg/v3 => ../client/pkg
-	go.etcd.io/etcd/client/v2 => ./../client/internal/v2
```

Comment:
you could lock this to v2.305.0 to keep the test

Reply:
something like `go.etcd.io/etcd/client/v2 => go.etcd.io/etcd/client/v2 v2.305.0`

Reply:
It's a good suggestion, but we need to fix & merge & backport https://github.com/etcd-io/etcd/pull/19675 firstly :)

Reply:
resolved in this PR & thx